src/opera/commands/package.py

```diff
@@ -0,0 +1,71 @@
+import argparse
+from pathlib import Path, PurePath
+import shutil
+import tempfile
+from zipfile import ZipFile
+
+from opera.error import DataError, ParseError
+from opera.parser import tosca
+from opera.storage import Storage
+
+
+def add_parser(subparsers):
+    parser = subparsers.add_parser(
+        "package",
+        help="Package service template and all accompanying files into a CSAR"
+    )
+    parser.add_argument(
+        "--output", "-O", type=argparse.FileType("w"),
```

Comment:
Just wondering - could option `-o` be better than `-O` capitalized? And the same question applies to `-L`. Because till now we always used small letters for the command options.

Reply:
This could be added as a future wish/feature.